app/tests/content/test_user_integration.py

```diff
@@ -469,3 +477,26 @@ def test_destroy_other_user_as_index_user(member, user, api_client):
     response = client.delete(url)
 
     assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
+def test_expired_user_event_shows_as_expired(member, api_client):
```

Comment:
Endre navn til å være mer likt konvensjoner for andre lignende tester. Et eksempel kan være noe slikt som:

test_list_expired_user_events

Viktigste er egt her at man bruker ordet list, siden man skal liste opp events